cdap-security-spi/src/main/java/io/cdap/cdap/security/spi/authorization/AuditLogContext.java

```diff
@@ -0,0 +1,35 @@
+/*
+ * Copyright © 2023 Cask Data, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not
+ * use this file except in compliance with the License. You may obtain a copy of
+ * the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations under
+ * the License.
+ */
+
+package io.cdap.cdap.security.spi.authorization;
+
+/**
+ * An interface to store the audit context received from chemist
+ */
+public interface AuditLogContext {
```

Comment:
I am wondering if we should provide a default "not required" implementation for this as a lot of extensions won't implement it

Reply:
This can achieved by using `new DefaultAuditLogContext()` , which will give `isAuditLoggingRequired` as false.